src/sql/workbench/browser/modelComponents/tree.component.ts

```diff
@@ -160,4 +171,8 @@ export default class TreeComponent extends ComponentBase implements IComponent,
 	public set withCheckbox(newValue: boolean) {
 		this.setPropertyFromUI<azdata.TreeProperties, boolean>((properties, value) => { properties.withCheckbox = value; }, newValue);
 	}
+
+	private onkeydown(domNode: HTMLElement, listener: (e: IKeyboardEvent) => void): void {
```

Comment:
seems not necessary to have this method. you can directly do this._regoster... there

Reply:
Eh, true. Just got lazy copying from other components. TBH I think it'd be better just to have this be on the ComponentBase class instead since there's going to be a bunch of components that use it. Thoughts on that?

Reply:
sounds like the right thing to do.